bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java

```diff
@@ -99,28 +101,15 @@
   private final Map<MethodDescriptor<?, ?>, Predicate<?>> methodsToRetryMap =
       createMethodRetryMap();
 
-  /**
-   * Creates a SslContext.
-   */
-  public interface SslContextFactory {
-    SslContext create();
-  }
+  private final static SslContextBuilder sslBuilder = GrpcSslContexts.forClient();
 
-  public static final SslContextFactory SSL_CONTEXT_FACTORY = new SslContextFactory() {
-    @SuppressWarnings("deprecation")
-    @Override
-    public SslContext create() {
-      try {
-        // We create multiple channels via refreshing and pooling channel implementation.
-        // Each one needs its own SslContext.
-        return SslContext.newClientContext();
-      } catch (SSLException e) {
-        throw new IllegalStateException("Could not create an ssl context.", e);
-      }
+  static {
```

Comment:
nit:
Can we make it a private static method?

private final static SslContextBuilder sslBuilder = getSslContextBuilder();

private static SslContextBuilder getSslContextBuilder() {
  SslContextBuilder sslBuilder = GrpcSslContexts.forClient();
  ...
  return sslBuilder;
}